test/SlowTests/Client/Subscriptions/ConcurrentSubscriptionsTests.cs

```diff
@@ -1232,10 +1232,12 @@ public async Task ConcurrentSubscriptionsShouldProcessWhen_2_ConnectionsAreSubsc
 
                 var db = await Databases.GetDocumentDatabaseInstanceFor(store);
                 var testingStuff = db.ForTestingPurposesOnly();
-
+                var connectionsCount = 0L;
```

Comment:
had to change this test since now both `SubscribeAsync` & `_subscriptionConnectionsState.InitializeAsync` are under same lock